src/nav/IconUnreadMentions.js

```diff
@@ -0,0 +1,39 @@
+/* @flow strict-local */
+
+import React, { PureComponent } from 'react';
+import { View } from 'react-native';
+
+import type { Dispatch } from '../types';
+import { BRAND_COLOR } from '../styles';
+import { connect } from '../react-redux';
+import { getUnreadMentionsTotal } from '../selectors';
+import { IconMention } from '../common/Icons';
+import { CountOverlay } from '../common';
+
+type Props = {|
+  dispatch: Dispatch,
+  unreadMentionsTotal: number,
+  color: string,
+|};
+
+class IconUnreadMentions extends PureComponent<Props> {
+  static defaultProps = {
+    color: BRAND_COLOR,
+  };
```

Comment:
Let's make this exactly parallel to `IconUnreadConversations`: the caller passes the color.

Reply:
Done 👍